<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPasswordEnty</class>
 <widget class="QDialog" name="DialogPasswordEnty">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add password entry</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password entry</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/passwprd_black.png</normaloff>:/icons_black/icons_black/passwprd_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Password</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>UUID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>URL:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="buttonPwdShow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/visible_black.png</normaloff>:/icons_black/icons_black/visible_black.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Notes:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Quality:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Group:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="textPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="buttonGenerator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Open password generator</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/password2_black.png</normaloff>:/icons_black/icons_black/password2_black.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="textUsername"/>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>128</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%v bits</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="buttonOpenUrl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Open URL</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/external_link2_black.png</normaloff>:/icons_black/icons_black/external_link2_black.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="QLineEdit" name="textUrl">
         <property name="inputMask">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>http://</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="4">
        <widget class="QTextEdit" name="textNotes"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="buttonAddIcon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resource.qrc">
           <normaloff>:/icons_black/icons_black/key2_black.png</normaloff>:/icons_black/icons_black/key2_black.png</iconset>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="4">
        <widget class="QComboBox" name="textGroup"/>
       </item>
       <item row="10" column="1" colspan="4">
        <widget class="QLineEdit" name="textTags"/>
       </item>
       <item row="11" column="1" colspan="4">
        <widget class="QLabel" name="labeluuid">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>UUID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="textTitle"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHistory">
      <attribute name="icon">
       <iconset resource="../../resources/resource.qrc">
        <normaloff>:/icons_black/icons_black/last6hours_black.png</normaloff>:/icons_black/icons_black/last6hours_black.png</iconset>
      </attribute>
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonArchive">
            <property name="toolTip">
             <string>Add to history</string>
            </property>
            <property name="text">
             <string>Bookmark</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resource.qrc">
              <normaloff>:/icons_black/icons_black/bookmark2_black.png</normaloff>:/icons_black/icons_black/bookmark2_black.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonRestore">
            <property name="text">
             <string>Restore</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resource.qrc">
              <normaloff>:/icons_black/icons_black/replace_black.png</normaloff>:/icons_black/icons_black/replace_black.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonRemoveFromArchive">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resource.qrc">
              <normaloff>:/icons_black/icons_black/delete_black.png</normaloff>:/icons_black/icons_black/delete_black.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="dateCreated">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::YearSection</enum>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Date created:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateTimeEdit" name="dateModified">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateTimeEdit" name="dateExpire">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="tableHistory">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>23</number>
         </attribute>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Date modified:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkExpire">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Date expire:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textTitle</tabstop>
  <tabstop>textUsername</tabstop>
  <tabstop>textPassword</tabstop>
  <tabstop>textUrl</tabstop>
  <tabstop>textNotes</tabstop>
  <tabstop>textTags</tabstop>
  <tabstop>buttonPwdShow</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPasswordEnty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPasswordEnty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
